src/native-common/FrontLayerViewManager.tsx

```diff
@@ -216,8 +216,11 @@ export class FrontLayerViewManager {
         return null;
     }
 
-    private _onBackgroundPressed = (e: RN.SyntheticEvent<any>) => {
-        e.persist();
+    private _onBackgroundPressed = (e: RN.GestureResponderEvent) => {
```

Comment:
I've added `persist` to `SyntheticEvent` interface - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26474. `RN.GestureResponderEvent` should contain `persist` now.